src/components/organisms/Feedbacks/FeedbackCard.jsx

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import Card, { variants as cardVariants } from '../../atoms/Card';
+
+const Photo = styled.div`
```

Comment:
It should be `picture` tag with `display: block;`.

All the magic with auto switching between avif/webp/jpg works only with `<picture>`